consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftController.java

```diff
@@ -158,8 +158,9 @@ private void handleMessage(final Message message) {
   public void handleNewBlockEvent(final NewChainHead newChainHead) {
     final BlockHeader newBlockHeader = newChainHead.getNewChainHeadHeader();
     final BlockHeader currentMiningParent = currentHeightManager.getParentBlockHeader();
+    LOG.debug("Handling New Chain head event, chain length = {}", currentMiningParent.getNumber());
```

Comment:
isn't this the chain height not chain length? elsewhere is logs I think block is just used to infer block number.

Reply:
Updated to be chain height.